docs/resources/addons-eks.yaml

```diff
@@ -0,0 +1,43 @@
+---
+
+# When deploying on AWS EKS, set the parameters, install aws-alb-ingress-controller and metrics-server
+# to true and then fill in the remaining parameters with your EKS environment details.
+aws-alb-ingress-controller:
```

Comment:
@taz-mon there are recent changes around AWS load balancer https://github.com/pegasystems/pega-helm-charts/blob/master/charts/addons/values.yaml#L37 which might require update here as well.

Reply:
Updated the addons-eks yaml file and the addons section of the EKS runbook to reflect latest changes in the addons file to support the use of an AWS load balancer ingress.